app/controllers/warehouse_reports/health/aco_performance_controller.rb

```diff
@@ -13,14 +13,43 @@ class AcoPerformanceController < ApplicationController
     before_action :require_can_view_aggregate_health!
     before_action :require_can_administer_health!
     before_action :set_aco, only: [:index]
+    before_action :set_report_year, only: [:index]
 
     def index
-      @report = Health::AcoPerformance.new(@aco) unless @aco.zero?
+      @report = Health::AcoPerformance.new(@aco, @report_year) unless @aco.zero?
     end
 
+    def report_periods
+      @report_periods = begin
+        periods = {
+          "9/2/#{Date.current.prev_year.year} to 9/1/#{Date.current.year}" => Date.current.prev_year.year,
```

Comment:
I guess this is ok, but this isn't consistent with the date format throughout the rest of the warehouse.